src/main/java/bio/terra/cli/serialization/userfacing/UFResource.java

```diff
@@ -34,7 +34,7 @@
     include = JsonTypeInfo.As.EXISTING_PROPERTY,
     property = "resourceType")
 @JsonSubTypes({
-  @Type(value = UFGcpNotebook.class, name = "AI_NOTEBOOK"),
+  @Type(value = UFGcpNotebook.class, name = "GCP_NOTEBOOK"),
```

Comment:
Let's revert this enum change. Changing the string here will break backwards compatibility.